docsrc/conf.py

```diff
@@ -160,4 +161,17 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# html_static_path = ['_static']
\ No newline at end of file
+# html_static_path = ['_static']
+
+typehints_use_signature = True
+typehints_use_signature_return = True
+autodoc_typehints = "both"
+autodoc_typehints_format = "short"
```

Comment:
Can you add a little comment that makes clear which sphinx module these are from?

Reply:
I'll fix it soon